AK/Assertions.cpp

```diff
@@ -77,6 +80,14 @@ ALWAYS_INLINE void dump_backtrace()
     free(syms);
 }
 }
+#elif __cpp_lib_stacktrace >= 202011L
+namespace {
+ALWAYS_INLINE void dump_backtrace()
+{
+    // We assume the stacktrace implementation demangles symbols, as does microsoft/STL
+    std::cerr << std::stacktrace::current() << "\n";
```

Comment:
Why are we using iostream here? Can't we `PRINT_ERROR(std::to_string(std::stacktrace::current()).c_str()); PRINT_ERROR("\n")` ?